addons/mail/static/src/scss/attachment_box.scss

```diff
@@ -2,6 +2,22 @@
 
     .o_mail_chatter_attachments {
         min-height: 50px;
+        position: relative;
+
+        .o_attachments_file_drop_zone {
```

Comment:
This is basically copy-paste of style from basic composer dropzone...
Thee correct way is to define a unique class that is shared between both

Reply:
Still unresolved

Reply:
@alexkuhn I still not resolved it, I will resolve it soon :)

Reply:
Ok, no problem :)